examples/flutter_gallery/.idea/workspace.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Oh, I get it now. Did not consider that adding these will cause git to think any further changes should be committed. I will remove them again.